src/Widget/Twig/TwigPreprocessor.php

```diff
@@ -316,14 +316,17 @@ public function preprocessEventDetail(Event $event, string $langcode, array $set
             $promotionsQuery->max = 4;
             $promotionsQuery->balieConsumerKey = $event->getOrganizer()->getCdbid();
             $promotionsQuery->unexpired = true;
+            $organizer = $this->translateOrganizerName($event, $langcode);
 
             try {
                 $uitpasPromotions = $this->cultureFeed->uitpas()->getPromotionPoints($promotionsQuery);
                 $variables['uitpas_promotions'] = $this->twig->render(
                     'widgets/search-results-widget/uitpas-promotions.html.twig',
                     [
                         'promotions' => $this->preprocessUitpasPromotions($uitpasPromotions),
-                        'organizer' => $this->translateOrganizerName($event, $langcode),
+                        'organizer' => $organizer,
```

Comment:
Nit: Might be a bit clearer if this was `organizerName`. But all in all these changes look good 👍

Reply:
jup! thx for the tip